backend/services/requestService.test.ts

```diff
@@ -0,0 +1,41 @@
+import { WhereaboutsApi } from '../api'
+import NotificationService from './notificationService'
+import RequestService from './requestService'
+import type { RequestEmail } from './model'
+
+jest.mock('../api')
+jest.mock('./manageCourtsService')
+
+const whereaboutsApi = new WhereaboutsApi(null) as jest.Mocked<WhereaboutsApi>
+const notificationService = new NotificationService(null, null, null) as jest.Mocked<NotificationService>
+
+describe('Request service', () => {
+  const context = {}
+  const user = 'user1'
+  const emailData = {} as RequestEmail
+  let service: RequestService
+
+  beforeEach(() => {
+    service = new RequestService(whereaboutsApi, notificationService)
+    notificationService.sendBookingRequestEmails = jest.fn()
```

Comment:
would generally jest.mock the notification service and it would create mocks for each method anyway